utils_linux.go

```diff
@@ -240,7 +241,15 @@ func (r *runner) run(config *specs.Process) (int, error) {
 			return -1, err
 		}
 	}
-	if r.detach {
+	if !r.create {
+		if err := process.Signal(syscall.SIGCONT); err != nil {
+			r.terminate(process)
+			r.destroy()
+			tty.Close()
+			return -1, err
+		}
+	}
+	if r.detach || r.create {
```

Comment:
On Tue, May 17, 2016 at 11:11:16AM -0700, Michael Crosby wrote:
> > +	if r.detach || r.create {
> 
> negative 

Can you unpack that a bit?  Why do you need to close the TTY in the
‘create’ case?  Why aren't you waiting on the container process and
returning it's error code?